examples/next-typescript/pages/checkbox.tsx

```diff
@@ -0,0 +1,102 @@
+import { Checkbox } from "@tonightpass/kitchen";
+import { NextPage } from "next";
+import React from "react";
+
+const CheckboxPage: NextPage = () => {
+  return (
+    <>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          justifyContent: "center",
+          height: "100vh",
+          gap: "10px",
+        }}
+      >
+        <p>default</p>
+        <div
+          style={{
+            display: "flex",
+            gap: "16px",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Checkbox>Default</Checkbox>
+        </div>
+
+        <p>checked</p>
+        <div
+          style={{
+            display: "flex",
+            gap: "16px",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Checkbox checked>Checked</Checkbox>
```

Comment:
Why when you click on the checkbox it gets unchecked it should stay checked when it is true and unchecked when it is false
Reference: https://vercel.com/design/checkbox#checked